io.openliberty.java.internal_fat_22/.gitignore

```diff
@@ -0,0 +1,6 @@
+.gradle/
+/bin/
+/build/
+/generated/
+/gradle/
```

Comment:
You should remove /gradle/ from the .gitignore.  
This is where the gradle-wrapper.jar and properties are stored.
Normally we don't persist jars but these files are what need to be persisted in order for the wrapper to work.